geth/jail/execution_policy.go

```diff
@@ -32,26 +31,25 @@ func NewExecutionPolicy(
 }
 
 // Execute handles the execution of a RPC request and routes appropriately to either a local or remote ethereum node.
-func (ep *ExecutionPolicy) Execute(req common.RPCCall, call otto.FunctionCall) (*otto.Object, error) {
+func (ep *ExecutionPolicy) Execute(req common.RPCCall, vm *vm.VM) (map[string]interface{}, error) {
 	if params.SendTransactionMethodName == req.Method {
-		return ep.executeSendTransaction(req, call)
+		return ep.executeSendTransaction(vm, req)
 	}
 
 	client := ep.nodeManager.RPCClient()
 
-	return ep.executeWithClient(client, req, call)
+	return ep.executeWithClient(client, vm, req)
 }
 
 // executeRemoteSendTransaction defines a function to execute RPC method eth_sendTransaction over the upstream server.
-func (ep *ExecutionPolicy) executeSendTransaction(req common.RPCCall, call otto.FunctionCall) (*otto.Object, error) {
-	res, err := call.Otto.Object(`({"jsonrpc":"2.0"})`)
-	if err != nil {
-		return nil, err
+func (ep *ExecutionPolicy) executeSendTransaction(vm *vm.VM, req common.RPCCall) (map[string]interface{}, error) {
+	// TODO(tiabc): Move it to the bottom where `result` and `hash` are set.
```

Comment:
Can we move it down in this PR? It seems like it can be just returned with all values in `return` statement.